roles/download/defaults/main.yml

```diff
@@ -60,7 +60,7 @@ downloads:
     url: "{{ calico_cni_download_url }}"
     owner: "root"
     mode: "0755"
-    enabled: "{{ kube_network_plugin == 'calico' or kube_network_plugin == 'canal' }}"
+    enabled: "{{ kube_network_plugin == 'calico' and not use_calico_cni_container|default(false) }}"
```

Comment:
Yep, in our CI :) But we'll switch to calico/cni container as soon as this PR is merged. So OK, I'll update this PR